packages/ui/src/ui/components/ACL/ACL.scss

```diff
@@ -77,16 +82,16 @@
     }
 
     .elements-table {
```

Comment:
It looks like ACL.js don't use ElementsTable anymore. Is it really need here?

Reply:
Yes, we don't need `.elements-table` style rules here anymore, I am going to remove them from here and update farm stands 👍🏻